src/components/MessageBanner/MessageBanner.ts

```diff
@@ -53,7 +53,7 @@ namespace fabric {
     /**
      * shows banner if the banner is hidden
      */
-    public showBanner(): void {
+    public show(): void {
```

Comment:
Could you keep `showBanner()` in place as a deprecated method? It can simply call `show()` for now.